### PR TITLE
Portal actionmenu overlay to direct child of body

### PR DESCRIPTION
**Pull request by twclark0**
## Instructions

- Action menu (specifically the overlay)

### What You're Solving

This is in response to: https://pluralsight.slack.com/archives/C70HPSJPP/p1582318454236600

As a side note, I wasn't able to get storybook running so I didn't actually verify that it works correctly. Can someone try it out on their machine and see if it puts the overlay just under the body?

### CODE REVIEW

File: packages/actionmenu/src/react/overlay.js [hunk 1]
```diff
@@ -11,21 +11,31 @@ const styles = {
   overlay: () => css(stylesheet['.psds-actionmenu__overlay'])
 }
 
+const hasComponentRendered = () =>
```
Comment (jaketrent) [inline packages/actionmenu/src/react/overlay.js]: Will this break SSR?  window is not defined outside the browser.

Comment (twclark0) [reply packages/actionmenu/src/react/overlay.js]: Right, so ideally line 16 catches that?

Comment (twclark0) [reply packages/actionmenu/src/react/overlay.js]: This is the same format done in the position package

Comment (danethurber) [reply packages/actionmenu/src/react/overlay.js]: https://github.com/JedWatson/exenv is the react core way to do this check

Comment (twclark0) [reply packages/actionmenu/src/react/overlay.js]: I have never seen this. It's a package last updated almost three years ago. 

You want me to update the window check with this package?

**Comment by danethurber** [comment]
merged

**Comment by twclark0** [comment]
Hey @danethurber what does versioning and publishing this package look like? So we know when we can update our package.

**Comment by danethurber** [comment]
@twclark0 sorry i missed you message. it's published. notes here

https://pluralsight.slack.com/archives/C70HQMTUM/p1582752534000100

**Pull request closed without merging**